src/sass/app_2/_common/base/constants.scss

```diff
@@ -26,7 +26,7 @@ $COLOR_LIGHT_BLACK  : #333;
 $COLOR_BLACK        : #000;
 
 $COLOR_GREEN        : #4bbfab;
-$COLOR_GRAY         : #7f8397;
```

Comment:
Isn't `$COLOR_GRAY` used in a lot of places throughout app2?

Reply:
there are two $COLOR_GRAY's actually, the duplicated added from one of the commit with master